engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobLifeCycleListener.java

```diff
@@ -36,21 +36,21 @@
      * Triggered when a node starts processing a task. This will be executed
      * on the nodes themselves.
      */
-    void onPartitionProcessingStart();
+    void onPartitionProcessingStart(SparkJobContext sparkJobContext);
```

Comment:
Yeah, I guess no problem... Though I don't where to get them. You can show me when you get here.